pyproject.toml

```diff
@@ -16,6 +16,8 @@ aiohttp = "^3"
 requests = "^2"
 click = "^8"
 pydantic = "<2"
+websocat = "^1.12.0"
+golem-node = "0.12.2"
```

Comment:
Reqc told me to use 0.13.1 as 12.3 has something for us (but it is not piped...)

Reply:
Yup, we are aware of that request, yet pushing new yagna version to the users, without significant testing from our side is not a good idea from my side.